src/CLI/CLI.php

```diff
@@ -445,6 +434,38 @@ public function prompt(string $prompt, bool $required = true)
 		return $value;
 	}
 
+	/**
+	 * Removes a folder including all containing files and folders
+	 */
+	public function rmdir($dir): bool
+	{
+		$dir = realpath($dir);
+
+		if (is_dir($dir) === false) {
+			return true;
+		}
+
+		if (is_link($dir) === true) {
+			return unlink($dir);
+		}
+
+		foreach (scandir($dir) as $childName) {
```

Comment:
Should we work here with 

```php
$iterator = new RecursiveDirectoryIterator($item, RecursiveDirectoryIterator::SKIP_DOTS);
 $children = new RecursiveIteratorIterator($iterator, RecursiveIteratorIterator::CHILD_FIRST);
 ```
 ?
 
 I have on my mind that they are more performant as `scandir` , but maybe I remember that wrong.

Reply:
Isn't this a recursive directory deletion? So I've copied `Dir::remove` method to use somewhere in CLI.

Reply:
Sure, nevermind.